ketcher-autotests/tests/utils/canvas/helpers.ts

```diff
@@ -77,6 +124,32 @@ export async function getCoordinatesTopAtomOfBenzeneRing(page: Page) {
   };
 }
 
+export async function getCoordinatesTopAtomOfCyclopentadieneRing(page: Page) {
```

Comment:
hmmm, we already have a function for this. May be we need to think about better name? Like
```
getCoordinatesOfTopMostCarbon(page: Page)
```
Then this function can be used for any structure

Reply:
Done.